src/concrete/ml/torch/numpy_module.py

```diff
@@ -43,7 +43,12 @@ def __init__(
                 dummy_input is not None
             ), "dummy_input must be provided if model is a torch.nn.Module"
 
-            self.numpy_forward, self._onnx_model = get_equivalent_numpy_forward_from_torch(
+            (
+                self.numpy_preprocessing,
```

Comment:
I think `numpy_preprocessing` should be private + have a getter as well ! 

also, as stated in https://github.com/zama-ai/concrete-ml/pull/778/files#r1670793432, I think we should provide better naming here in order to make sure we do not confuse these with the QM's attribute(s)

Reply:
why would it be more private than numpy_forward?

Reply:
I updated with your suggestion